is_core/generic_views/inlines/__init__.py

```diff
@@ -1,22 +1,57 @@
 from django.views.generic.base import ContextMixin
 from django.template.loader import render_to_string
+from django.utils.translation import ugettext as _
+
+from is_core.auth.views import FieldPermissionViewMixin
+from is_core.auth.permissions import IsAdminUser, FieldsSetPermission
+
+from .permissions import RelatedCoreAllowed
 
 
 class InlineView(ContextMixin):
+
     template_name = None
 
+    permission = IsAdminUser()
+
     def __init__(self, request, parent_view, parent_instance):
         self.request = request
         self.parent_view = parent_view
         self.parent_instance = parent_instance
+        assert self.permission, 'Permissions must be set'
 
     def render(self, context, title):
-        template = self.template_name
-        context.update(self.get_context_data(title=title))
-        return render_to_string(template, context.flatten())
+        if self.permission.has_permission('read', self.request, self):
+            template = self.template_name
+            context.update(self.get_context_data(title=title))
+            return render_to_string(template, context.flatten())
+        else:
+            return ''
```

Comment:
inline view now check its permission which can get from related_core (core of the model)